vividus-extension-selenium/src/main/java/org/vividus/steps/ui/validation/IBaseValidations.java

```diff
@@ -111,5 +120,14 @@ boolean assertIfExactNumberOfElementsFound(String businessDescription, SearchCon
 
     List<WebElement> assertIfAtLeastNumberOfElementsExist(String businessDescription, Locator locator, int leastNumber);
 
+    /**
+     * Asserts that an element located by the <b>locator</b> does not exists
+     *
+     * @param businessDescription The description of the element being validated, must not be {@code null}.
+     * @param locator     The locator to find element, must not be {@code null}.
+     * @return true if element doesn't and false otherwise.
+     * @deprecated Use {@link #assertElementDoesNotExist(String, Locator)}
+     */
```

Comment:
```suggestion
    /**
     * Asserts that an element located by the <b>locator</b> does not exist
     *
     * @param businessDescription The description of the element being validated, must not be {@code null}.
     * @param locator     The locator to find element, must not be {@code null}.
     * @return true if element does not exist and false otherwise.
     * @deprecated Use {@link #assertElementDoesNotExist(String, Locator)}
     */
```